<launch>
  <!-- Run the costmap node -->
  <node name="costmap_node" pkg="costmap_2d" type="costmap_2d_node" output="screen">
	    <rosparam file="/home/rahul/AGV/hybrid_astar_workspace/src/hybrid_astar/launch/gazebo_params.yaml" command="load" ns="costmap" />  
  </node>

</launch>


